app/views/registration_commands/_form.erb

```diff
@@ -48,28 +48,36 @@
   </div>
   <div class='form-group'>
     <label class='col-md-2 control-label' for='setup_insights'>
-      <% help = _('If selected, Insights client will be installed and registered on Red Hat family operating systems. It has no effect on other OS families that do not support it.') %>
+      <% help = _('If selected, Insights client will be installed and registered on Red Hat family operating systems. It has no effect on other OS families that do not support it. Default value is defined by `host_registration_insights` global parameter.') %>
```

Comment:
```
If set to "Yes", Insights client will be installed and registered on Red Hat family operating systems. It has no effect on other OS families that do not support it.
The default value is based on the
`host_registration_insights` parameter. It can be inherited e.g. from host group, operating system, organization. When overidden, the selected value will be stored on host parameter level.
```